README.md

```diff
@@ -252,6 +252,16 @@ Alternatively you can assign a secret to the node to allow it to pull from your
 
 For Docker Swarm use the `--send-registry-auth` flag or its shorthand `-a` which will look up your registry credentials in your local credentials store and then transmit them over the wire to the deploy command on the API Gateway. Make sure HTTPS/TLS is enabled before attempting this.
 
+### Use faas-cli in CI environments
+
+If you're running faas-cli in a CI environment, chances are you get an env var as `CI=true` or `CI=1`, like [Github Actions](https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables), [CircleCI](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables), or [Travis](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables).
```

Comment:
"if you're running in a CI environment chances are"

Can you be more direct?

When running faas-cli in a CI service such as X, Y, Z, the CI environment variable is set to `true` by the platform.

Reply:
Resolved